src/App.jsx

```diff
@@ -1,13 +1,18 @@
 import "./App.css";
+import React from 'react';
 import TodoList from "./components/TodoList.jsx";
 import AddTodoForm from "./components/AddTodoForm.jsx";
 
 function App() {
+
+  const [newTodo, setNewTodo] = React.useState('');
```

Comment:
Importing useState directly  `import { useState } from 'react'; ` makes the code cleaner and easier to read by removing repeated React. prefixes. It aligns with modern React practices and helps identify exactly which hooks are being used in a component.